src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -713,13 +713,24 @@ public ResponseEntity<Mono<Void>> modifyLoad(@PathVariable("studyUuid") UUID stu
     @Operation(summary = "update a load creation in the study network")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The load creation has been updated")})
     public ResponseEntity<Mono<Void>> updateLoadCreation(@PathVariable("studyUuid") UUID studyUuid,
-                                                 @PathVariable("modificationUuid") UUID modificationUuid,
-                                                 @PathVariable("nodeUuid") UUID nodeUuid,
-                                                 @RequestBody String createLoadAttributes) {
+                                                         @PathVariable("modificationUuid") UUID modificationUuid,
+                                                         @PathVariable("nodeUuid") UUID nodeUuid,
+                                                         @RequestBody String createLoadAttributes) {
         return ResponseEntity.ok().body(studyService.assertComputationNotRunning(nodeUuid)
                 .then(studyService.updateEquipmentCreation(studyUuid, createLoadAttributes, ModificationType.LOAD_CREATION, nodeUuid, modificationUuid)));
     }
 
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/modifications/{modificationUuid}/loads-modification")
```

Comment:
do we really want 1 path for each modification ? 
can't we have /studies/{studyId}/nodes/{...}/network-modifications/{modId}/{type-modification}
where type-modification is an enum ?

Reply:
I agree this could be done with a single path. A dedicated PR should be opened in order to modify all currently existing modifications paths to use a single path.